control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -60,6 +60,56 @@ def __getattr__(self, name: str) -> object:
+
+    @property
+    def persistence_mode(self) -> str:
+        return self.inner.persistence_mode
+
+    def transaction(self, *, isolation_level: str | None = None):
+        return self.inner.transaction(isolation_level=isolation_level)
+
+    def get(self, record_type: object, record_id: str) -> object | None:
+        record = self.inner.get(record_type, record_id)
+        if (
+            record_type is ActionRequestRecord
```

Ruff (0.15.12)

[error] Undefined name `ActionRequestRecord`

(F821)